### PR TITLE
Update translation jp kr

### PR DESCRIPTION
**Pull request by Thunderbird2086**
### Check if this PR fulfills these requirements:
* [x] My changes respect the [Kodi add-on development rules](https://kodi.wiki/view/Add-on_rules)
* [x] I have read the [**CONTRIBUTING**](Contributing.md) document.
* [x] I made sure there wasn't another one [Pull Requests](../../pulls) opened for the same update/change
* [x] I have successfully tested my changes locally

#### Types of changes
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Feature change (non-breaking change which change behaviour of an existing functionality)
- [x] Improvement (non-breaking change which improve functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Description
1. Update Korean Translation
1. Add Japanese Translation

#### Describe the current behavior
Showing only English regardless of Language settings

#### Describe the new behavior
Showing proper language as per settings

### CODE REVIEW

File: resources/language/resource.language.ja_jp/strings.po [hunk 1]
```diff
@@ -0,0 +1,965 @@
...
+msgid "Do you have a Netflix Premium account?"
+msgstr "Netflixプレミアムアカウントをお持ちですか。"
+
+msgctxt "#30156"
+msgid "Does your device support the 4K standard?\r\nThe device must:\r\nBe Netflix certified, support for Widevine Security level L1 and have a 4K display HDCP 2.2"
+msgstr "このデバイスは4K標準を従っていますか。\r\nこのデバイスは必ず:\r\nNetflixの認証を受け、WidevineセキュリティレベルL1対応や4KディスプレHDCP2.2が必要です。"
+
+msgctxt "#30157"
+msgid "The InputStream Adaptive settings panel will now be opened, so you have to change this option \"Override HDCP status\" to ON"
+msgstr "インプットストリーム設定画面が開いたら、\"Override HDCP statu\"を ONにしてください"
```
Comment (CastagnaIT) [inline resources/language/resource.language.ja_jp/strings.po]: missing "s" in "status"

File: resources/language/resource.language.ja_jp/strings.po [hunk 2]
```diff
@@ -0,0 +1,965 @@
...
+msgid "Show trailers & more"
+msgstr "予告編など"
+
+msgctxt "#30180"
+msgid "There has been a problem with login, it is possible that your account has not been confirmed or renewed."
+msgstr "ログイン失敗、アカウント確認がまだ取れてないか更新出来なかった可能性があります。"
+
+msgctxt "#30181"
+msgid "Disable subtitles if there are no forced subtitles streams (Works only if Kodi Player is set to \"Only forced\" subtitles)"
+msgstr "強制字幕のない場合、字幕を表示しない(Kodiプレイヤーが\"強制字幕のみ\"の場合"
```
Comment (CastagnaIT) [inline resources/language/resource.language.ja_jp/strings.po]: missing ")" at the end?

File: resources/language/resource.language.ko_kr/strings.po [hunk 3]
```diff
@@ -314,8 +322,644 @@ msgstr "새로 고칠 때 몇분 마다 확인할까요?"
...
+msgid "Restrict by Maturity Level"
+msgstr "연령제한"
+
+msgctxt "#30233"
+msgid "Content for {} will require the PIN to start playback."
+msgstr "{}을(를) 보려면 PIN을 입력하십시오."
+
+msgctxt "#30234"
+msgid "Keep watched status marks separate for each profile (this option will be removed in future versions)"
+msgstr "    "
```
Comment (CastagnaIT) [inline resources/language/resource.language.ko_kr/strings.po]: forgot?

**Comment by CastagnaIT** [comment]
also modifies the file addon.xml
so in the tag languages:
add "ja ko"
betweeen: "it" and: "nl"

File: resources/language/resource.language.ko_kr/strings.po (hunk 3 above)
Comment (Thunderbird2086) [reply resources/language/resource.language.ko_kr/strings.po]: Probably I forgot to commit my last change 😉

**Comment by CastagnaIT** [comment]
tag <language> still missing

**Comment by CastagnaIT** [comment]
very good thanks

**Pull request merged**